Dockerfile.ci

```diff
@@ -3,16 +3,17 @@ FROM elifesciences/proofreader-php:latest AS proofreader
 FROM elifesciences/annotations_cli:${commit}
 USER root
 
-# extract into another image
-RUN git clone https://github.com/asm89/smoke.sh /opt/smoke.sh
-
 RUN mkdir -p build && chown www-data:www-data build
 
 USER elife
 COPY --from=proofreader --chown=elife:elife /srv/proofreader-php /srv/proofreader-php
 RUN ln -s /srv/proofreader-php/bin/proofreader /srv/bin/proofreader
 
+USER root
+RUN composer --no-interaction install --no-suggest
```

Comment:
CI should be fine with `--classmap-authoritative` too, makes it a bit faster.

Reply:
Base test classes seem not being picked up 
```
Fatal error: Class 'tests\eLife\Annotations\WebTestCase' not found in /srv/annotations/tests/Annotations/AnnotationsTest.php on line 10
script returned exit code 255
```
from https://alfred.elifesciences.org/blue/organizations/jenkins/pull-requests-projects%2Fannotations/detail/PR-71/13/pipeline

Reply:
Can reproduce locally `docker-compose -f docker-compose.ci.yml run --rm ci ./project_tests.sh`

Reply:
Seems to be missing from the class map:
```
www-data@1096fc52059f:/srv/annotations$ grep -r WebTestCase vendor/composer/    
vendor/composer/autoload_static.php:        'Silex\\WebTestCase' => __DIR__ . '/..' . '/silex/silex/src/Silex/WebTestCase.php',
vendor/composer/autoload_classmap.php:    'Silex\\WebTestCase' => $vendorDir . '/silex/silex/src/Silex/WebTestCase.php',
```

Reply:
Regenerating manually the class map sees it appear:
```
root@9ea8d6e777ad:/srv/annotations# grep -r WebTestCase vendor/composer/
vendor/composer/autoload_static.php:        'Silex\\WebTestCase' => __DIR__ . '/..' . '/silex/silex/src/Silex/WebTestCase.php',
vendor/composer/autoload_static.php:        'tests\\eLife\\Annotations\\WebTestCase' => __DIR__ . '/../..' . '/tests/Annotations/WebTestCase.php',
vendor/composer/autoload_classmap.php:    'Silex\\WebTestCase' => $vendorDir . '/silex/silex/src/Silex/WebTestCase.php',
vendor/composer/autoload_classmap.php:    'tests\\eLife\\Annotations\\WebTestCase' => $baseDir . '/tests/Annotations/WebTestCase.php',
```